src/SFA.DAS.Tasks.API.Client/ITaskAPIClient.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
 using SFA.DAS.Tasks.API.Types.DTOs;
+using SFA.DAS.Tasks.API.Types.Enums;
 
 namespace SFA.DAS.Tasks.API.Client
 {
     public interface ITaskApiClient
     {
-        Task<IEnumerable<TaskDto>> GetTasks(string employerAccountId, string userid);
+        Task<IEnumerable<TaskDto>> GetTasks(string employerAccountId, string userid, ApprenticeshipEmployerType applicableToApprenticeshipEmployerType);
```

Comment:
Why make a breaking change to the interface?  Can a default value be provided or an overload?

Reply:
There were two options, one was to add the extra parameter (which I have done) and the other was to have a completely separate endpoint to get Reminder tasks as they cannot be determined without knowing the employer type. Either way a breaking change was inevitable.

Reply:
can we default to both ? so until we change the calling code it would continue to work as previous?

Reply:
Can do.

Reply:
can we default to both ? so until we change the calling code it would continue to work as previous?